app/javascript/find/controllers/radius_quick_link_suggestions_controller.js

```diff
@@ -34,37 +33,27 @@ export default class extends Controller {
         if (!response.ok) throw new Error('Network response was not ok')
         return response.json()
       })
-      .then(data => {
-        this.renderLinks(data)
-      })
+      .then(data => this.renderLinks(data))
       .catch(error => {
         console.error('Fetch error:', error)
-        this.containerTarget.innerHTML = ''
+        this.loadingTarget.innerHTML = ''
       })
   }
 
   renderLinks (links) {
+    this.loadingTarget.innerHTML = ''
+
     if (!links.length) {
       this.containerTarget.innerHTML = ''
-      this.headingTarget.innerHTML = ''
       return
     }
 
-    const subjectName = this.subjectNameValue
-    const headingText = subjectName
-      ? `Try browsing for '${subjectName}' in a wider location search`
-      : 'Try browsing with a wider location search'
-
-    this.headingTarget.innerHTML = `<h3 class="govuk-heading-m">${headingText}</h3>`
-
-    const listItems = links.map(link =>
-      `<li class="govuk-list govuk-list--bullet"><a href="${link.url}" class="govuk-link">${link.text}</a></li>`
-    ).join('')
+    const template = document.getElementById('radius-suggestions-template').innerHTML
+    const rendered = Mustache.render(template, {
+      subjectName: this.subjectNameValue,
+      links: links
+    })
```

Comment:
Let's use object shorthand where possible. I'll update our linter rules in a future PR.

```suggestion
    const rendered = Mustache.render(template, {
      subjectName: this.subjectNameValue,
      links
    })
```